src/shared/components/case-viewer/case-viewer.component.spec.ts

```diff
@@ -603,15 +618,6 @@ describe('CaseViewerComponent', () => {
     expect(orderService.sort).toHaveBeenCalledWith(FIELDS);
   });
 
-  it('should render the event log component in the event log tab', () => {
```

Comment:
This is no longer on the case viewer template level. It's tested down in the case-history-viewer level.